unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ChatController/ChatController.cs

```diff
@@ -216,11 +225,11 @@ public void MuteChannelError(string payload)
     public void UpdateChannelSearchResults(string payload)
     {
         var msg = JsonUtility.FromJson<ChannelSearchResultsPayload>(payload);
-        var channelsResult = new Channel[msg.channels.Length];
+        var channelsResult = new Channel[msg.channels.channelInfoPayload.Length];
```

Comment:
the data structure is wong. This is the rfc declaration:
```
channelSearchResultsPayload: {
    since:string? // nullable pagination token
    channels: channelInfoPayload
}
```
msg.channels should be the list of channels information

Reply:
Fixed in https://github.com/decentraland/unity-renderer/pull/3145/commits/0beb6790fb6aa4274b8b60978f60e1a943047a8a